chords.py

```
@@ -337,6 +345,8 @@ def main():
         parser.print_help()  # Print help if no options are selected
         return
 
+    baudrate = args.baudrate  # Get baud rate from arguments
```

Ruff (0.8.2)

Local variable `baudrate` is assigned to but never used

Remove assignment to unused variable `baudrate`

(F841)